synthtool/languages/node.py

```diff
@@ -260,3 +262,58 @@ def postprocess_gapic_library_hermetic(hide_output=False):
     fix_hermetic(hide_output=hide_output)
     compile_protos_hermetic(hide_output=hide_output)
     logger.debug("Post-processing completed")
+
+
+def owlbot_main():
+    """Copies files from staging and template directories into current working dir.
+
+    When there is no owlbot.py file, run this function instead.  Also, when an
+    owlbot.py file is necessary, the first statement of owlbot.py should probably
+    call this function.
+
+    Depends on owl-bot copying into a staging directory, so your .Owlbot.yaml should
+    look a lot like this:
+
+        docker:
+            image: gcr.io/repo-automation-bots/owlbot-nodejs:latest
+
+        deep-remove-regex:
+            - /owl-bot-staging
+
+        deep-copy-regex:
+            - source: /google/cloud/video/transcoder/(.*)/.*-nodejs/(.*)
+              dest: /owl-bot-staging/$1/$2
+
+    Also, this function requires a default_version in your .repo-metadata.json.  Ex:
+        "default_version": "v1",
+    """
+    logging.basicConfig(level=logging.DEBUG)
+    staging = Path("owl-bot-staging")
+    s_copy = transforms.move
+    if staging.is_dir():
+        # Load the default version defined in .repo-metadata.json.
+        default_version = json.load(open(".repo-metadata.json", "rt"))[
+            "default_version"
+        ]
+        # Collect the subdirectories of the staging directory.
+        versions = [v.name for v in staging.iterdir() if v.is_dir()]
+        # Reorder the versions so the default version always comes last.
+        versions = [v for v in versions if v != default_version] + [default_version]
+
+        # Copy each version directory into the root.
+        for version in versions:
+            library = staging / version
```

Comment:
Looks like `transforms.move` already did what we needed?

Reply:
Yep.  It was called `s.copy()` in synth.py files.